packages/e2e/features/ui/components/authenticator/sign-in-with-phone.feature

```diff
@@ -75,17 +75,6 @@ Feature: Sign In with Phone Number
     And I click the "Sign out" button
     Then I see "Sign in"
 
-
-  # FORCE_CHANGE_PASSWORD tests are skipped as the temporary passwords used for these
```

Comment:
These tests have since been then added to `sign-in-force-new-password` tests.

Reply:
👍